Multiplatform/NowPlaying/Regular/RegularNowPlayingView.swift

```diff
@@ -47,7 +46,7 @@ struct RegularNowPlayingView: View {
                                 }
                                 .transition(.blurReplace)
                             }
-                            .frame(width: UIScreen.main.bounds.width / 2)
```

Comment:
The queue / lyrics should take up half the screen, but only when the app is not in split view / vertical orientation. But for this to work I have to add the iOS now playing layout first, so this should work in the meantime

Reply:
You are right, it just happens to kind of align on the iPad I am using.